packages/localizations/src/sv-SE.ts

```diff
@@ -511,51 +514,56 @@ export const svSE: LocalizationResource = {
     },
     start: {
       actionLink: 'Logga in',
-      actionLink__use_email: undefined,
-      actionLink__use_phone: undefined,
+      actionLink__use_email: 'Använd e-post istället',
+      actionLink__use_phone: 'Använd telefon istället',
       actionText: 'Har du redan ett konto?',
       subtitle: 'för att fortsätta till {{applicationName}}',
       title: 'Skapa ditt konto',
     },
+    restrictedAccess: {
+      title: 'Begränsad åtkomst',
+      subtitle: 'Åtkomst till denna app är begränsad och en inbjudan krävs för att registrera sig.',
+      actionLink: 'Tillbaka till inloggning',
+    },
   },
   socialButtonsBlockButton: 'Fortsätt med {{provider|titleize}}',
-  socialButtonsBlockButtonManyInView: undefined,
+  socialButtonsBlockButtonManyInView: '{{provider|titleize}}',
   unstable__errors: {
-    already_a_member_in_organization: undefined,
+    already_a_member_in_organization: '{{email}} är redan medlem i organisationen.',
     captcha_invalid:
       'Registrering misslyckades på grund av säkerhetskontroller. Vänligen uppdatera sidan och försök igen eller kontakta supporten för mer hjälp.',
     captcha_unavailable:
       'Registrering misslyckades på grund av misslyckad bot-validering. Vänligen uppdatera sidan och försök igen eller kontakta supporten för mer hjälp.',
-    form_code_incorrect: 'Koden är felaktig.',
-    form_identifier_exists: 'Identifieraren finns redan.',
-    form_identifier_exists__email_address: 'E-postadressen finns redan.',
-    form_identifier_exists__phone_number: 'Telefonnumret finns redan.',
-    form_identifier_exists__username: 'Användarnamnet finns redan.',
-    form_identifier_not_found: 'Identifieraren hittades inte.',
-    form_param_format_invalid: 'Formatet är ogiltigt.',
+    form_code_incorrect: undefined,
+    form_identifier_exists: undefined,
+    form_identifier_exists__email_address: 'Denna e-postadress är taget. Vänligen prova ett annat.',
+    form_identifier_exists__phone_number: 'Detta telefonnummer är taget. Vänligen prova ett annat.',
+    form_identifier_exists__username: 'Detta användarnamn är taget. Vänligen prova ett annat.',
+    form_identifier_not_found: undefined,
+    form_param_format_invalid: undefined,
```

Comment:
form_code_incorrect can be "koden är felaktig" still and the undefined ones can revert as they were since they are correct. The ones I changed are more descriptive with "please try somehting else", so if that is fine they can be changed to the new format otherwise they can be reverted.